packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -54,8 +55,8 @@ const MT5AccountCard = ({
                 {icon && <IconComponent />}
                 <div className='mt5-account-card__type--description'>
                     <h1 className='mt5-account-card--heading'>{title}</h1>
-                    {!existing_data && <p className='mt5-account-card--paragraph'>{descriptor}</p>}
-                    {existing_data && existing_data.display_balance && (
+                    {(!existing_data || !is_logged_in) && <p className='mt5-account-card--paragraph'>{descriptor}</p>}
```

Comment:
which description is this?

Reply:
<img width="326" alt="Screenshot 2020-07-16 at 2 28 47 PM" src="https://user-images.githubusercontent.com/45410833/87634879-b975ef80-c770-11ea-834e-72db7a70c33c.png">

Reply:
Apparently it goes missing upon login and logout...so i added that flag